docs/settings/search-sessions-settings.asciidoc

```diff
@@ -0,0 +1,31 @@
+
+[[search-session-settings-kb]]
+=== Search sessions settings in {kib}
+++++
+<titleabbrev>Search sessions settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use {kib} search session features. They are
+enabled by default.
+
+
+[cols="2*<"]
+|===
+| `search:sessions:enabled`
+  | Set to `true` (default) to enable search sessions.
+
+| `search:Sessions:trackingInterval`
+  | The frequency for updating the state of a search session. The default is 10s.
+
+| `search:Sessions:pageSize`
+  | The number of search sessions to show per page in *Stack Management*. The default is 10000.
```

Comment:
I see we don't list all of the options here (which is good, some of them are exposed for troubleshooting purposes).
I think the `pageSize` shouldn't be here. Especially as it refers to how many search sessions we fetch from Elasticsearch and not the actual number of docs we show per page.